ffi-ctests/ctests/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.2)
```

Comment:
Cmake seems a bit of overkill for a single file. Maybe just use `cc` crate in the build script directly?